modules/Expectation.js

```diff
@@ -364,6 +365,42 @@ class Expectation {
     return this
   }
 
+  toIncludeKeys(keys, comparator, message) {
+    if (typeof comparator === 'string') {
+      message = comparator
+      comparator = null
+    }
+
+    containKeysHelper(
```

Comment:
Same as in #86, I'd like to see the branching inline here instead of tucked away in the `containKeysHelper` function, please.